src/PSAppDeployToolkit/Public/Invoke-ADTCommandWithRetries.ps1

```diff
@@ -70,6 +87,10 @@ function Invoke-ADTCommandWithRetries
         [ValidateRange(1, 60)]
         [System.UInt32]$SleepSeconds = 5,
 
+        [Parameter(Mandatory = $false)]
```

Comment:
Can you put this into a separate parameter set? That'd be the cleaner way than allowing all parameters, but having one parameter override another with `$PSBoundParameter.ContainsKey()` checks.

Reply:
It's not meant to completely override the `-Retries` parameter, just when that parameter isn't provided.